independent-projects/resteasy-reactive/client/runtime/src/main/java/org/jboss/resteasy/reactive/client/impl/ClientBuilderImpl.java

```diff
@@ -66,8 +66,8 @@ public class ClientBuilderImpl extends ClientBuilder {
     private String nonProxyHosts;
 
     private boolean followRedirects;
-    private boolean trustAll;
-    private boolean verifyHost;
+    private boolean trustAll = false;
```

Comment:
This is false by default, no need to change it

Reply:
Fixed in https://github.com/quarkusio/quarkus/pull/32932/commits/7babb3221397a336d33cf7c155642230878d97fc